src/routes/welcome/gettingStarted.tsx

```diff
@@ -6,23 +6,37 @@ import browser from "webextension-polyfill";
 
 import styled from "styled-components";
 
-import Completed from "./gettingStarted/completed";
-import Enabled from "./gettingStarted/enableNotifications";
-import Connect from "./gettingStarted/connect";
-import { useLocation } from "wouter";
-import Explore from "./gettingStarted/explore";
-
-const gettingStartedPages = [
-  <Completed />,
-  <Enabled />,
-  <Explore />,
-  <Connect />
+import { CompletedWelcomeView } from "./gettingStarted/completed";
+import { EnableNotificationsWelcomeView } from "./gettingStarted/enableNotifications";
+import { ConnectWelcomeView } from "./gettingStarted/connect";
+import { ExploreWelcomeView } from "./gettingStarted/explore";
+import { useLocation } from "~wallets/router/router.utils";
+import type { CommonRouteProps } from "~wallets/router/router.types";
+import { Redirect } from "~wallets/router/components/redirect/Redirect";
+
+const Views = [
```

Comment:
This could/should be replaced with `<Routes>` too. See https://github.com/arconnectio/ArConnect/issues/567.